packages/components/addon/components/hds/stepper/stepper-indicator/index.hbs

```diff
@@ -0,0 +1,22 @@
+<div class={{this.classNames}} ...attributes>
```

Comment:
issue (a11y, blocker): This will need some sort of role, depending on the intent of the design.